doc/src/records/dhwheater.md

```diff
@@ -85,6 +85,28 @@ available for electric (air source heat pump and resistance) SMALLSTORAGE water
   ----------- ---------------------- ------------- -------------- -----------------
                *Codes listed above*   FUEL          No             constant
 
+**whResType=*choice***
+
+Resistance heater type, valid only if wh_heatSrc=_ELRESX, else ignored. These choices are supported by the detailed HPWH model.  Except for Generic, all heater characteristics are set by HPWH based on whResType.
```

Comment:
Some of the comments in CNRECS refer to other CNRECS field names, but in the user documentation they should refer to other CUL names.